lib/bundle/brew_installer.rb

```diff
@@ -2,6 +2,7 @@ module Bundle
   class BrewInstaller
     def self.install(name, options = {})
       new(name, options).install_or_upgrade
+      BrewServices.restart(name) if options[:restart_service]
```

Comment:
The `if` can be postfix and you don't need the `else` (as you want to return it unconditionally)